src/state/trajectory/reducer.ts

```diff
@@ -37,7 +38,7 @@ export const initialState = {
     timeUnits: null,
     scaleBarLabel: "",
     agentIds: [],
-    agentUiNames: [],
+    defaultUIData: [],
```

Comment:
is this still an empty array? I don't remember the shape of UIData, but null might make more sense if UIData isn't an array

Reply:
UIDisplayData is an array.

Reply:
I remember we disagreed about this once before, but I do wonder if reducer/initialState would be easier to understand at a glance with more explicit typing?

Reply:
it's not a problem as long as this is the correct empty state